<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>164</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="line_edit_precipitation">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_edit_hw">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>precipitation (mm/h)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>hw (m/h)</string>
   </property>
  </widget>
  <widget class="QSlider" name="slider_hours">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>48</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_time">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>hours</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_time_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>90</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>Excel File</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>time</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>slider_hours</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_time_2</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>slider_hours</sender>
   <signal>valueChanged(int)</signal>
   <receiver>Dialog</receiver>
   <slot>rain_infiltration()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>53</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>line_edit_precipitation</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Dialog</receiver>
   <slot>check_slider()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>219</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>33</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>time_update()</slot>
  <slot>slot1()</slot>
  <slot>slot2()</slot>
  <slot>rain_infiltration()</slot>
  <slot>check_slider()</slot>
 </slots>
</ui>
